docs/content/v0.20/examples.md

```diff
@@ -25,7 +25,7 @@ The following steps establish an initial state used in the examples below.
    ```
 
 3. Create an inventory & mailbox space of type `vcluster` running *OCM* (Open Cluster Management)
-in KubeFlex. Note that `-p ocm` runs a post-create hook on the *vcluster* control plane 
```

Comment:
I do not understand why GitHub is saying that there is a trailing space to be deleted here. This PR was rebased onto `main` after that space was deleted, right?